### PR TITLE
Add Dutch passive voice

### LINKED ISSUE #1017: Add passive voice for Dutch
We should add the passive voice assessment for Dutch.

### PR DESCRIPTION
**Pull request by manuelaugustin**
## Summary

This PR can be summarized in the following changelog entry:

* Adds the passive voice assessment for Dutch.

## Test instructions

This PR can be tested by following these steps:

* Build a beta script using `trunk` for Yoast SEO and this branch for YoastSEO.js.
* Copy and paste a Dutch text from e.g. Wikipedia. See if passive voice is detected (if there are no passive sentences, this might also be due to the text; try a different text in that case).
* Make sure the following kinds sentences are marked as passive:
  * Passive sentences with the auxiliary `worden`, e.g. `Een appel wordt gegeten`.
* Make sure the following kinds of sentences are _not_ marked as passive:
  * Non-passive sentences, e.g. `Ik eet een appel.`

Fixes #1017

### CODE REVIEW

File: spec/researches/passiveVoice/matchParticiplesDutchSpec.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	} );
+
+	it( "does not match a word that is not a regular participle", function() {
+		expect( matchParticiples.regularParticiples( "banaan", "nl" ) ).toEqual( [] );
+	} );
+
+	it( "does not match an empty string", function() {
+		expect( matchParticiples.regularParticiples( "", "nl" ) ).toEqual( [] );
+	} );
+} );
```
Comment (nataliashitova) [inline spec/researches/passiveVoice/matchParticiplesDutchSpec.js]: Please add specs for irregularParticiples as well.

File: src/researches/passiveVoice/periphrastic/getSentencePartsSplitOnStopwords.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+/**
+ * Splits up the sentence in parts based on stopwords.
+ *
+ * @param {string} sentence The sentence to split up in parts.
+ * @param {string} language The language for which to split sentences into parts.
+ * @returns {Array} The array with the sentence parts.
+ */
+module.exports = function( sentence, language ) {
```
Comment (igorschoester) [inline src/researches/passiveVoice/periphrastic/getSentencePartsSplitOnStopwords.js]: Why is this a wrapper function and not just export the splitSentence?

**Comment by igorschoester** [comment]
Acceptance 🚧 
I built a beta with trunk and it got me a sprintf error `[sprintf] expecting number but found string`.
Then I built it with the branch of https://github.com/Yoast/wordpress-seo/pull/10411 and it got me the same error.

Then I noticed the filenames of 10411 are not correct now. Therefor this was an exercise in futility 😞

File: spec/researches/dutch/passiveVoice/DutchParticipleSpec.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+		const mockParticiple = new DutchParticiple( "achtervolgd", "werd achtervolgd.", { auxiliaries: [ "werd" ], type: "irregular", language: "nl" } );
+		expect( mockParticiple.getParticiple() ).toBe( "achtervolgd" );
+		expect( mockParticiple.isOnNonParticiplesList() ).toBe( false );
+		expect( mockParticiple.hasNonParticipleEnding() ).toBe( false );
+		expect( mockParticiple.directPrecedenceException( mockParticiple._sentencePart, 5, "nl" ) ).toBe( false );
+		expect( mockParticiple.determinesSentencePartIsPassive() ).toBe( true );
+	} );
+
+	it( "checks the properties of the Dutch participle object with a word from the list of non-participles", function() {
+		const mockParticiple = new DutchParticiple( "beschrijvend", "wordt beschrijvend.", { auxiliaries: [ "werd" ], type: "regular", language: "nl" } );
```
[the remark this answers is not in the document]
Comment (manuelaugustin) [reply spec/researches/dutch/passiveVoice/DutchParticipleSpec.js]: Fixed

**Pull request merged**